### PR TITLE
release-notes: add initial release notes for the v94 release

### PR DESCRIPTION
**Pull request by ricardosalveti**
# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Release notes for the v94 release.

## Checklist

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by ricardosalveti** [comment]
Still missing the aktualizr release notes, but wanted to open the pr to check if I'm adding to the right location :-)

**Comment by ricardosalveti** [comment]
For ak I think this is the release notes, from jira:

- Offline update bundles are now tied to a specific tag and device type. This is enforced by a new signed metadata file added to the bundle by fioctl >= 0.42.
- Applications can be included in offline update bundles by using new factory configuration parameters (containers.offline.enable and containers.offline.app_shortlist), instead of having to enable image preloading. See [Offline - Updates — FoundriesFactory 93 documentation](https://docs.foundries.io/latest/user-guide/offline-update/offline-update.html#prerequisites) for mode details.
- Update events and callbacks are now enabled during offline updates.
- Applications state verification, removal and pruning are now performed by the composectl tool.
- Bug fixes.

@detsch @mike-sul ^^

**Comment by doanac** [comment]
Docs for 240c4cda7f00f4a728550ab6efb8a94ce8b7e3e6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2696/docs/artifacts/html/index.html

**Comment by kprosise** [comment]
> Still missing the aktualizr release notes, but wanted to open the pr to check if I'm adding to the right location :-)

yep!

File: release-notes/rn_v94.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# LmP V93 Release Notes
```
Comment (kprosise) [inline release-notes/rn_v94.md:1]:
```suggestion
# LmP V94 Release Notes
```

**Review by kprosise — APPROVED** [review]
Looking good!

**Comment by detsch** [comment]
> For ak I think this is the release notes, from jira:
> 
> * Offline update bundles are now tied to a specific tag and device type. This is enforced by a new signed metadata file added to the bundle by fioctl >= 0.42.
> * Applications can be included in offline update bundles by using new factory configuration parameters (containers.offline.enable and containers.offline.app_shortlist), instead of having to enable image preloading. See [Offline - Updates — FoundriesFactory 93 documentation](https://docs.foundries.io/latest/user-guide/offline-update/offline-update.html#prerequisites) for mode details.
> * Update events and callbacks are now enabled during offline updates.
> * Applications state verification, removal and pruning are now performed by the composectl tool.
> * Bug fixes.
> 
> @detsch @mike-sul ^^

I wrote that based on a internal list of changes Mike Sul sent on Slack. @mike-sul can you take a look just to make sure no relevant information was lost in the process?

**Comment by mike-sul** [comment]
> * fioctl >= 0.42

They are seem fine. However I get 404 when I try to open the v94 release notes on the docs generated by this PR.

File: release-notes/rn_v94.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+> [!NOTE]
+  You can view the full [v94 changelog](https://foundries.io/products/releases/94/) for greater detail.
+
+* [v94 Test Results](https://qa-reports.foundries.io/lmp/lmp-ci-testing/build/e27cb36c6c8df52df543887cdb6b407ce84ef33d/)
+
+## Attention Points for Migration
+
+Things to be aware of when [updating LmP](https://docs.foundries.io/94/reference-manual/linux/linux-update.html)from the v93 release:
```
Comment (quaresmajose) [inline release-notes/rn_v94.md:10]: missing space before ` from`

**Comment by doanac** [comment]
Docs for 82ec0f6263fd50c487bf2e0e5b2ba23633f98581 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2700/docs/artifacts/html/index.html

File: release-notes/rn_v94.md (hunk 2 above)
Comment (ricardosalveti) [reply release-notes/rn_v94.md]: Fixed

**Comment by doanac** [comment]
Docs for 10c22b2a6289b2aeb8c5b36e33000a7d28b2f455 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2701/docs/artifacts/html/index.html

**Comment by ricardosalveti** [comment]
@kprosise good from my side, feel free to merge if fine by you.

**Pull request merged**